app/views/layouts/editorial.html.haml

```diff
@@ -1,7 +1,7 @@
 !!! 5
 
 %html.js
-  = render 'head', title: 'GovAuBeta: Collaboration'
+  = render 'head', title: (content_for?(:title) ? "#{yield(:title)} | GOV.AU Collaboration" : 'GOV.AU Collaboration' )
```

Comment:
I know it was previously Collaboration, but should the title just be Editorial? This is the only place we call it Collaboration. The breadcrumbs, title etc is all Editorial.